.asyncapi-tool.yaml

```
@@ -0,0 +1,10 @@
+title: AsyncAPI Server API
+links:
+  websiteUrl: https://api.asyncapi.com/v1
+  docsUrl: https://api.asyncapi.com/v1/docs
+filters:
+  technology:
+    - Node js
+    - TypeScript
+  categories:
+    - api
```

```suggestion
title: AsyncAPI Server API
links:
  websiteUrl: https://api.asyncapi.com/v1
  docsUrl: https://api.asyncapi.com/v1/docs
filters:
  technology:
    - Node.js
    - TypeScript
  categories:
    - api
```

suggesting the change only to get this merged and try to trigger CI runs again